### PR TITLE
Modify lib/stale.js

### PR DESCRIPTION
**Pull request by apb7**
This file has been modified to fetch pull request author in markComment section of the stale.yml file.
(At the moment, I already have a fork of the original repo in my account. I will create a branch on my own fork and then send in a pull request from the next time).

### CODE REVIEW

File: lib/stale.js [hunk 1]
```diff
@@ -108,13 +108,15 @@ module.exports = class Stale {
     const {owner, repo} = this.config
     const perform = this.getConfigValue(type, 'perform')
     const staleLabel = this.getConfigValue(type, 'staleLabel')
-    const markComment = this.getConfigValue(type, 'markComment')
+    var markComment = this.getConfigValue(type, 'markComment')
```
Comment (kevinlee12) [inline lib/stale.js]: Let's move this below the constants since it's a different type of variable.

Comment (apb7) [reply lib/stale.js]: Shouldn't we place all `this.getConfigValue` statements together first?

Comment (kevinlee12) [reply lib/stale.js]: It doesn't affect the value, so no?

Comment (apb7) [reply lib/stale.js]: Done, thanks!

**Review by kevinlee12 — APPROVED** [review]
lgtm!

**Pull request merged**